src/pages/index.tsx

```diff
@@ -1,8 +1,25 @@
+import { GetServerSideProps, GetStaticProps, NextPage } from "next";
 import Head from "next/head";
-import styled from "styled-components";
+import PositionCard from "../components/modules/IdeaRecruitPage/PositionCard";
 import FilteringComponent from "../components/modules/MainPage/filtering/FilteringComponent";
+import { Project } from "../components/modules/MainPage/filtering/project";
 
-const IndexPage = () => (
+export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
getServerSideProps gets called every time someone navigates to this route, in which I think it's an overkill. please use getStaticProps with revalidate property

Reply:
Consider it done